.rubocop.yml

```diff
@@ -0,0 +1,82 @@
+require:
+  - rubocop-minitest
+  - rubocop-performance
+
+AllCops:
+  NewCops: disable
+  Exclude:
+    - "bin/**/*"
+    - "Gemfile"
+    - "Gemfile.lock"
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+Layout/EndOfLine:
+  EnforcedStyle: lf
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/EmptyLinesAroundAccessModifier:
+  EnforcedStyle: only_before
+
+Layout/LineLength:
+  Exclude:
+    - "scripts/generate_fixture.rb"
+
+Lint/SuppressedException:
+  Exclude:
+    - "test/**/*"
+
+Lint/Void:
+  Exclude:
+    - "lib/generate_representation.rb"
```

Comment:
I added this exclusion due to [this line](https://github.com/exercism/ruby-representer/blob/master/lib/generate_representation.rb#L8). Let me know if this is correct.

Reply:
I think [both these lines](https://github.com/exercism/ruby-representer/blob/master/lib/generate_representation.rb#L7-L8) should be deleted. They look like they've been accidentally pasted in. This rubocop violation is actually catching an error I think, so I think it should the exclusion should be removed and the code fixed.

Reply:
Aha, nice! I figured it was some fancy Ruby metaprogramming :D